### PR TITLE
Update FPGA documentation

### PR DESCRIPTION
**Pull request by gpsimenos**
This PR updates the command to run FPGA tests in documentation and adds info about the `FPGA_FORCE_ALL_PORTS` macro.

### CODE REVIEW

File: docs/porting/target/static_pinmap.md [hunk 1]
```diff
@@ -169,7 +169,9 @@ Total flash memory (text + data): 44554(-1010) bytes
 Run FPGA tests to check whether your implementation is valid:
 
 ```
- mbed test -t ARM -m K64F -n tests-mbed_hal_fpga_ci_test_shield*
+mbed test -n "hal-tests-tests*fpga*" --app-config TESTS/configs/fpga.json
 ```
 
+The `FPGA_FORCE_ALL_PORTS` macro can be defined to force all peripherals to be tested. Some FPGA tests only test one by default, to save time.
+
```
Comment (jeromecoutant) [inline docs/porting/target/static_pinmap.md:180]: Add 
```
mbed test -n hal-tests-tests*fpga* --app-config TESTS/configs/fpga.json -DFPGA_FORCE_ALL_PORTS
```
as example ?

Comment (gpsimenos) [reply docs/porting/target/static_pinmap.md:180]: Done

**Comment by gpsimenos** [comment]
@jeromecoutant Do you think we can expand on what FPGA_FORCE_ALL_PORTS does? Marcelo thinks it's not clear enough.

**Comment by jeromecoutant** [comment]
If you propose to do so, I will agree :-)

**Review by MarceloSalazar — APPROVED** [review]

**Comment by MarceloSalazar** [comment]
@donatieng can we merge this?

**Review by donatieng — APPROVED** [review]

**Pull request merged**